addons/arsenal/stringtable.xml

```diff
@@ -433,5 +433,17 @@
             <Italian>Impossibile aprire l'arsenale ACE</Italian>
             <Japanese>ACE 武器庫を開けません</Japanese>
         </Key>
+        <Key ID= "STR_ACE_Arsenal_portLoadoutsText">
+            <English>import BI VA loadouts to ACE Arsenal</English>
+            <French>Importe les loadouts de BI VA dans ACE Arsenal</French>
+        </Key>
+        <Key ID= "STR_ACE_Arsenal_portLoadoutsPlayerError">
+            <English>No player unit available!</English>
```

Comment:
I would add instruction on what to do to make it work, such as `No player unit available! Place a unit and mark it as "Player"` or something.

Reply:
huh, surprisingly you can fit a lot more characters than I thought in those messages